src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -1,18 +1,26 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { connect } from 'react-redux';
-
-import { handleVerify } from '../../common/helpers/login-helpers.js';
 
 class AuthApplicationSection extends React.Component {
   constructor(props) {
     super(props);
-    this.handleVerify = handleVerify;
+    this.handleVerify = this.handleVerify.bind(this);
+  }
+
+  handleVerify() {
```

Comment:
This looks identical to the `handleVerify` function in `VerifyPrompt`, save for the prop with the url being slightly different. Is there a reason you removed this from the common helpers and put it in both files directly instead?